doc/classes/HSplitContainer.xml

```diff
@@ -13,6 +13,9 @@
 		<theme_item name="autohide" data_type="constant" type="int" default="1">
 			Boolean value. If 1 ([code]true[/code]), the grabber will hide automatically when it isn't under the cursor. If 0 ([code]false[/code]), it's always visible.
 		</theme_item>
+    <theme_item name="minimum_grab_thickness" data_type="constant" type="int" default="6">
```

Comment:
The indent is wrong and inconsistent. Did you add this manually? Make sure to run the doctool.

Reply:
I guess I modified this line by mistake. It should be fixed now.